source/deployment/ldap-group-sync.rst

```diff
@@ -0,0 +1,146 @@
+
+.. image:: ../images/Team_Channel_Membership_Sync.png
+
+Remove configured teams and channels from a group
+-------------------------------------------------
+To remove a team or channel configured for a group, click “Remove” to the right of the team or channel name. 
+
+View users belonging to the group
+---------------------------------
+
+Users who have logged in and accessed Mattermost will be visible in the members list on the group object. Members are read-only at this time and new members can be added through management in your AD/LDAP system. 
+
+.. image:: ../images/Group_Members.png
+
+Users can be removed from the group on subsequent synchronizations. However, they will not be removed from teams and channels. If a user is removed from a AD/LDAP group and then later re-added, they will be defaulted again into the teams and channels configured in the group. 
+
+.. note:: 
+When a member removes themselves manually from a channel, that action is tracked in the Channel Member History table.  Users are not re-added to channels from which they previously removed themselves. 
+
+Managing groups
+---------------
+Once a group has been configured, it can be edited to change the default teams and channels by clicking “Edit” on the group on the list view.  
+
+Deleting groups
+---------------
+Groups can be deleted by adjusting your AD/LDAP group filter to remove the group or by unlinking the group on the groups listing page. If you add the group back by re-adjusting the AD/LDAP group filter and link the group again on the group configuration page, the previous configurations will be available.
+
+Frequently Asked Questions
+----------------------------
+
+Why is AD/LDAP Group Sync in alpha?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We want to make sure we have tested this feature in environments that have different AD/LDAP system and group structures to ensure the feature works well in the different varieties of environments our customers have. Our testing has included Active Directory and Open LDAP systems. 
+
+When will AD/LDAP Group Sync be in beta?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We expect AD/LDAP Group Sync to be in beta early in 2019. 
+
+LDAP group sync will be in beta for a period of time until: 
+
+1. Searching and support for managing large volumes of groups and users is released. 
+2. This feature has been tested on a system with 10,000s of concurrent active users.
+
+Why can't my existing users see the teams and channels they have been synced to?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The next scheduled synchronization job that is run after the configuration of a group will add the teams and channels for any existing users.  You can manually initiate a synchronization from **System Console > Authentication > AD/LDAP > AD/LDAP Synchronize Now**.  
```

Comment:
0/5 - Could this be rephrased so the wording is more directly answering the question? 

Eg "Your users will see the teams and channels after....."

Reply:
updated.